contracts/bridge/BridgeHandledRequests.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.4.24;
 
-contract HandledRequests {
+contract BridgeHandledRequests {
```

Comment:
@KimKyungup Will you inherit this contract to other contracts other than `BridgeTransferCommon`?

Reply:
All KLAY/ERC20/ERC721 should have it. So `BridgeTransferCommon` inherits `BridgeHandledRequests`.

Reply:
If BridgeTransferCommon is a single parent, how about merging the `BridgeHandledRequests`?

Reply:
As you know, we have a worry that `handledRequests` is so heavy.
It just increase the storage. So maybe we need to optimize it or make it as optional feature. 
So I separated it. :)

Reply:
In this issue, I will follow the problem and improve this feature. :) Thanks for your comment.
https://groundx.atlassian.net/jira/software/projects/TB/boards/36/backlog?assignee=5b30eefd2213f42d59c1474f&selectedIssue=TB-408

Reply:
It will be okay if additional methods are added in the future.